File: OneDrive/Documents/Unam}/Ciclo-III/Analisis y diseños de algoritmos/Informe/Bubblesort.py
def Bubblesort(lista):
    for i in range(0,len(lista)-1):
        for j in range(0, len(lista)-1):
            if(lista[j]>=lista[j+1]):
                lista[j],lista[j+1]=lista[j+1],lista[j]
    return lista

File: OneDrive/Documents/Unam}/Ciclo-III/Analisis y diseños de algoritmos/Informe/bubblesort.cpp
#include <iostream>
using namespace std;
#include <stdio.h>
void swap(int *a, int *b) 
    { 
        int temp = *a; 
        *a = *b; 
        *b = temp; 
    }
void BubbleSort(int lista[],int len) 
    { 
        int i, j; 
        for (i = 0; i < len-1; i++)       
            for (j = 0; j < len-i-1; j++)  
                if (lista[j] > lista[j+1]) 
                    swap(&lista[j], &lista[j+1]);
    }
